conda/recipes/dask-cuda/meta.yaml

```diff
@@ -16,11 +16,16 @@ source:
 build:
   number: {{ GIT_DESCRIBE_NUMBER }}
   string: py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - dask-cuda-worker = dask_cuda.cli.dask_cuda_worker:go
+    - dask-config = dask_cuda.cli.dask_config:go
```

Comment:
Why do we have to specify this stuff twice (here and in pyproject.toml)? I guess it's because conda doesn't know anything about setuptools/pyproject? But given we're explicitly installing with pip should that not be fine?